app/models/query/last_updated_applications.rb

```diff
@@ -0,0 +1,11 @@
+module Query
+  class LastUpdatedApplications
+    def initialize(user)
+      @user = user
+    end
+
+    def find
+      @user.applications.order(updated_at: :desc).limit(5)
```

Comment:
This `.limit()` may cause confusion when using this class. 

If it is necessary then how about something like this:
```
def find(limit: nil)
  @user.applications.order(updated_at: :desc).limit(limit)
end
```
and then 
- `Query::LastUpdatedApplications.new.find` will return all the last updated applications
- `Query::LastUpdatedApplications.new.find(limit: 10)` will return the last 10 updated applications

Reply:
yeah make sense